package.json

```diff
@@ -4,14 +4,16 @@
   "scripts": {
     "ng": "ng",
     "start": "node server.js",
-    "serve": "ng serve --configuration=dev",
-    "start:staging": "gulp set --env=staging && concurrently --kill-others \"node server.js\" \"npm run serve\"",
-    "start:prod": "gulp set --env=prod && concurrently --kill-others \"node server.js\" \"npm run serve\"",
+    "start:staging": "gulp set --env=staging && concurrently --kill-others \"node server.js\" \"npm run serve:staging\"",
+    "start:prod": "gulp set --env=prod && concurrently --kill-others \"node server.js\" \"npm run serve:production\"",
```

Comment:
@Nishanthh You can replace the content in  `start:` scripts with the one in `serve:` scripts and remove the `serve:` scripts, we won't be using them

Reply:
@Nishanthh The port value can be given in angular.json file